src/Types/Conversions/ros/sensor_msgs/PointCloud2.hpp

```diff
@@ -12,6 +14,26 @@ namespace RosConversion {
         sensor_msgs::convertPointCloud2ToPointCloud(pointcloud, pcloud);
         convert(pcloud, to);
     }
+    
+    static void convert(const robot_remote_control::PointCloud &pointcloud, sensor_msgs::PointCloud2* to) {
```

Comment:
Any reason why you are not using the [sensor_msgs::convertPointCloudToPointCloud2](http://docs.ros.org/en/indigo/api/sensor_msgs/html/namespacesensor__msgs.html#a4e302e86e3c57ba454daa9f0bced8b21) function similar to the opposite conversion?

Reply:
This would still require someone to implement `static void convert(const robot_remote_control::PointCloud &pointcloud, sensor_msgs::PointCloud* to)` then, wouldn't it?

Reply:
true, didn't know that also does not exitst in https://github.com/dfki-ric/robot_remote_control/blob/master/src/Types/Conversions/ros/sensor_msgs/PointCloud.hpp 

But I think it whould be the cleaner approach to have it the same here. But still, performance wise i guess the proposed conversion is the better way performance wise, better than two conversions